eveonline/views.py

```diff
@@ -93,21 +93,41 @@ def api_sso_validate(request, tokens, api_id):
         auth, c = AuthServicesInfo.objects.get_or_create(user=request.user)
         if not auth.main_char_id:
             return redirect('auth_characters')
-        return redirect('auth_api_key_management')
+        return redirect('auth_dashboard')
     else:
         messages.warning(request, '%s not found on API %s. Please SSO as a character on the API.' % (token.character_name, api.api_id))
     return render(request, 'registered/apisso.html', context={'api':api})
 
 
 @login_required
 def api_key_management_view(request):
-    logger.debug("api_key_management_view called by user %s" % request.user)
+    logger.debug("DEPRECIATED api_key_management_view called by user %s" % request.user)
+    # Legacy redirect in case some links still ghost here
+    return redirect('auth_dashboard')
+
+
+@login_required
+def dashboard_view(request):
+    logger.debug("dashboard_view called by user %s" % request.user)
+    auth_info = AuthServicesInfo.objects.get_or_create(user=request.user)[0]
+    apikeypairs = EveManager.get_api_key_pairs(request.user.id)
+    sso_validation = settings.API_SSO_VALIDATION or False
+    api_chars = []
+
+    if apikeypairs:
+        for api in apikeypairs:
+            api_chars.append({
+                'id': api.api_id,
+                'sso_verified': api.sso_verified if sso_validation else True,
+                'characters': EveManager.get_characters_by_api_id(api.api_id),
+            })
+
     context = {
-        'apikeypairs': EveManager.get_api_key_pairs(request.user.id),
-        'api_sso_validation': settings.API_SSO_VALIDATION or False
+        'main': EveManager.get_character_by_id(auth_info.main_char_id),
```

Comment:
Why use the staticmethod here when you can just get the EveCharacter model directly?